ui/app/AppLayouts/Communities/popups/TransferOwnershipPopup.qml

```diff
@@ -1,93 +1,153 @@
+        CustomText {
+            text: qsTr("I acknowledge that...")
         }
 
-        StatusBaseText {
-            Layout.fillWidth: true
+        StatusCheckBox {
+            id: ackCheckBox
 
-            text: qsTr("You can also use this key to import your community on another device")
-            wrapMode: Text.WordWrap
-            font.pixelSize: 13
-            color: Theme.palette.baseColor1
+            Layout.topMargin: -Style.current.halfPadding
+            Layout.bottomMargin: Style.current.halfPadding
+
+            font.pixelSize: Style.current.primaryTextFontSize
+            text: qsTr("My ownership rights will be removed and transferred to the recipient")
         }
     }
 
+    header: StatusDialogHeader {
+        headline.title: qsTr("Transfer ownership of %1").arg(root.communityName)
+        leftComponent: Rectangle {
+            height: 40
+            width: height
+            radius: height/2
+            color:Theme.palette.baseColor2
+
+            Image {
+                id: image
+
+                source: root.communityLogo
+                anchors.fill: parent
+                fillMode: Image.PreserveAspectFit
+                visible: false
+            }
+
+            OpacityMask {
+                anchors.fill: image
+                source: image
+                maskSource: parent
+            }
+        }
+        actions.closeButton.onClicked: root.close()
+    }
+
     footer: StatusDialogFooter {
-        leftButtons: ObjectModel {
-            StatusBackButton {
+        spacing: Style.current.padding
+        rightButtons: ObjectModel {
+            StatusButton {
+                text: qsTr("Cancel")
+                normalColor: "transparent"
+
                 onClicked: {
-                    root.close()
+                    root.cancelClicked()
+                    close()
+                }
+            }
+
+            StatusButton {
+                enabled: ackCheckBox.checked
+                text: qsTr("Send %1 owner token").arg(root.communityName)
+                type: StatusBaseButton.Type.Danger
+
+                onClicked: {
+                    // Pre-populated dialog with the relevant Owner token info:
+                    sendModal.sendType = Constants.SendType.Transfer
+
+                    sendModal.selectedAccount =  ModelUtils.getByKey(root.accounts, "address", token.accountAddress)
+                    sendModal.preSelectedHolding = {
+                        uid : token.key,
+                        chainId: token.chainId,
+                        name: token.name,
+                        imageUrl: token.artworkSource,
+                        collectionUid: "",
+                        collectionName: ""
+                    }
+                    sendModal.preSelectedHoldingType = Constants.HoldingType.Collectible
+                    sendModal.open()
+                    close()
                 }
             }
         }
     }
-}
 
+    SendModal {
```

Comment:
Hmm, quite heavy; what about wrapping it in a `Component` and open it using `Global.openPopup()` instead?

Reply:
There is one instance in `AppMain`. I can use it instead! It was really my first though.. so yeah, I agree! :)